src/pages/StudyPage.js

```diff
@@ -167,17 +171,40 @@ class StudyPage extends React.Component {
                 />
 
                 <DownloadSVGPlot
+                  left={
+                    <MultiSelect
+                      value={this.state.focusChromosome}
+                      options={[
+                        { label: 'All chromosomes', value: '' },
+                        ...chromosomeNames.map(d => ({
+                          label: `Chromosome ${d}`,
+                          value: d,
+                        })),
+                      ]}
+                      handleChange={this.handleChange}
+                    />
+                  }
                   loading={loading}
                   error={error}
                   svgContainer={manhattanPlot}
                   filenameStem={`${studyId}-independently-associated-loci`}
                 >
-                  <ManhattanWithTooltip data={manhattan} ref={manhattanPlot} />
+                  <ManhattanWithTooltip
+                    data={manhattan}
+                    focusChromosome={this.state.focusChromosome}
+                    handleChromosomeClick={this.handleChromosomeClick}
+                    ref={manhattanPlot}
+                  />
                 </DownloadSVGPlot>
                 <ManhattanTable
                   loading={loading}
                   error={error}
-                  data={manhattan.associations}
+                  data={manhattan.associations.filter(
+                    d =>
+                      this.state.focusChromosome
+                        ? d.chromosome === this.state.focusChromosome
+                        : true
+                  )}
```

Comment:
How about here we do this:

```jsx
<ManhattanTable
  loading={loading}
  error={error}
  data={this.state.focusChromosome ? manhattan.associations.filter(d => d.chromosome === this.state.focusChromosome) : manhattan.associations}
  studyId={studyId}
/>
```
So that we avoid looping when we want to show all the chromosomes.